src/components/__tests__/FormRegisterCoordinator.cy.js

```diff
@@ -148,12 +153,12 @@ describe('<FormRegisterCoordinator>', () => {
       cy.dataCy('form-register-coordinator-responsibility')
         .find('.q-checkbox__bg')
         .invoke('height')
-        .should('be.equal', 18);
+        .should('be.equal', 16);
```

Comment:
The value 16 is repeated in the tests, you can use a global var with the corresponding name for this var.

Reply:
Fixed.